### DISCUSSION TITLE
很多issue实际上已经解决或已有结论，但是一直处于未关闭状态，是否考虑集中清理一波，让更需要解决的问题暴露出来

### DISCUSSION THREAD

**Discussion by CherishTheYouth** [element-plus/element-plus#18777] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `N/A`

## Description

很多issue实际上已经解决或已有结论，但是一直处于未关闭状态，是否考虑集成清理一波，让更需要解决的问题暴露出来

### Additional comments

很多issue实际上已经解决或已有结论，但是一直处于未关闭状态，是否考虑集成清理一波，让更需要解决的问题暴露出来

**Comment by thinkasany** [comment]
@btea 是不是可以搞一个钉钉群，结合webhook和ci转发issue、pr到群里，让活跃的开发者一起答疑或是讨论或分配任务～

**Reply by btea** [reply]
如 @CherishTheYouth 所说，有相关的discord频道，在同一个地方交流比较好，也不需要额外的维护其他平台。

由于工作和生活的原因，一些维护人员也逐渐难以分散精力在这方面，导致我们在discord平台活跃性比较低。如果你们有兴趣的话，欢迎在discord上进行相关问题讨论，有问题也可以@我，我会尽力回复。（由于要工作，可能回复不及时，请见谅。）

**Comment by CherishTheYouth** [comment]
之前好像有 discard 群，现在不活跃了

**Comment by shizhihuaxu** [comment]
确实，好多挺关键的 pr 一直没有被合，新的 pr 出来后都被淹没了
